pkg/ddc/alluxio/transform.go

```diff
@@ -154,6 +154,14 @@ func (e *AlluxioEngine) transformCommonPart(runtime *datav1alpha1.AlluxioRuntime
 	if len(runtime.Spec.JvmOptions) > 0 {
 		value.JvmOptions = runtime.Spec.JvmOptions
 	}
+	// Alluxio v2.6.2 in container requires an extra jvmOption to specify # of cpus
+	if strings.Contains(runtime.Spec.AlluxioVersion.ImageTag, "2.6.2") {
+		if len(value.JvmOptions) > 0 {
+			value.JvmOptions = append(value.JvmOptions, "-XX:ActiveProcessorCount=8")
+		} else {
+			value.JvmOptions = []string{"-XX:ActiveProcessorCount=8"}
+		}
```

Comment:
Why is the active processor count  8?

Reply:
Some properties in Alluxio depends on this value, and since in container environment java can't get the correct number of cpus, 8 is a default value for calculating those Alluxio properties.

Reply:
How the user customize that? For example if  the ActiveProcessorCount is 16, what the user can do.

Further more, I think it will be better to calculate it in the alluxio entrypoint shell. It can get the cpucores of the virtual machine, and set the how many of them are used by Alluxio.

Reply:
Users can explicitly set the related property keys. The cpu number is just for default values. 

In terms of permanent fix, I'll discuss with the team about it. This definitely can be a way.